src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -153,13 +153,34 @@ def try_run_vipin(output_paths: OutputPaths) -> None:
         logger.warning(f"Appending performance data to central logs failed with: {e}")
 
 
+def write_backup_metadata(
```

Comment:
Is this what we were talking about that could be really slow say if we have 50k jobs?

Reply:
I still think it is fine if we only do this once.